pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "quartz_solar_forecast"
+version = "1.0.69"
+description = "Open Source Solar Forecasting for a Site"
+authors = [
+    { name = "Peter Dudfield", email = "info@openclimatefix.org" }
+]
+readme = "README.md"
+readme-content-type = "text/markdown"
+requires-python = ">=3.6"
+license = { text = "MIT" }
+dependencies = [
+    # Add dependencies from your requirements.txt here, e.g.
```

Comment:
could you add the dependecies, perhaps copies these from the requierments file, and remove the file